Documentation/contributing/development/dev_setup.rst

```diff
@@ -37,6 +37,21 @@ Detailed Instructions
 Depending on your specific development environment and requirements, you
 can follow the detailed instructions below.
 
+Using ``devcontainer``
```

Comment:
Can you move the below to the existing devcontainer section? 

https://docs.cilium.io/en/latest/contributing/development/dev_setup/#dev-container

https://github.com/cilium/cilium/blob/0ac34f084f726e638e75a9a639c5fd1acc6f9897/Documentation/contributing/development/dev_setup.rst?plain=1#L603

Reply:
Ah, sorry i missed that. that is much better. fixed in https://github.com/cilium/cilium/pull/31534/commits/e4b8b7a2708a039446af44d41061abc997a51e22